examples/filters.rs

```diff
@@ -112,9 +118,9 @@ impl RequestLogger {
 
 impl ContextFilter for RequestLogger {
     ///Count requests and log the path.
-    fn modify(&self, ctx: FilterContext, context: &mut Context) -> ContextAction {
+    fn modify(&self, _ctx: FilterContext, context: &mut Context) -> ContextAction {
         *self.counter.write().unwrap() += 1;
-        ctx.log.note(&format!("Request #{} is to '{}'", *self.counter.read().unwrap(), context.uri));
+        info!("Request #{} is to '{}'", *self.counter.read().unwrap(), context.uri);
```

Comment:
A simple line that says something like "Run this example with the environment variable 'RUST_LOG' set to 'debug' to see the debug prints." would probably be enough.